python/agents/medical-pre-authorization/medical_pre_authorization/prompt.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def return_root_agent_instruction() -> str:
```

Comment:
What is the purpose of this being a function?

Why can't we just have it be a string and then import it directly? 

Although it should probably be all capitals to signify a constant (i.e. `AGENT_INSTRUCTION`)

https://github.com/google/adk-samples/blob/8faee5f027ff454120da2f54e4449f4ffd077227/python/agents/software-bug-assistant/software_bug_assistant/prompt.py#L15